Ecommerce/app/views/layouts/_navbar.html.erb

```diff
@@ -17,15 +17,15 @@
 
         <li class="nav-item dropdown">
           <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
-          Categories
+            Categories
```

Comment:
why extra tab here?

Reply:
It is written inside the anchor tag that's why I added a tab space to show that it's inside anchor tag

Reply:
Okay, Thank you